src/streamlink/plugins/bongacams.py

```diff
@@ -138,13 +129,18 @@ def _get_streams(self):
                                        self._get_stream_uid(stream_source_info['userData']['username'])),
             # Multiple args with same name not supported.
             # Details: https://github.com/streamlink/streamlink/issues/321
-            "conn" : "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
+            "conn": "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
                 username=stream_source_info['userData']['username'],
                 access_key=stream_source_info['localData']['NC_AccessKey'],
                 data_key=stream_source_info['localData']['dataKey']
             )
         }
 
+        subprocess_errorlog_path = self.session.options.options.get('subprocess-errorlog-path')
```

Comment:
As a result, it would be acceptable following:
```python
if self.session.options.get('subprocess-errorlog-path') or self.session.options.get('subprocess-errorlog'):
    if self.session.logger.level > 3 and self._supports_param("debug"): # don't think '_supports_param' worth checking
        self.params["debug"] = True
    elif self._supports_param("verbos"):
        self.params["verbose"] = True
```